### DISCUSSION TITLE
Update organization with logo?

### DISCUSSION THREAD

**Discussion by ghost** [jnode-revisited/jnode#4] [Q&A] [answered]
Hi.

I would like to make a request to update the organization icon to the jnode logo.

#### img *before*
![image](https://user-images.githubusercontent.com/111701513/208280983-b414afcc-c33e-4a1b-82ff-f276d613dd6e.png)

#### img *after*
![image](https://user-images.githubusercontent.com/111701513/208280968-a552ac43-38e2-41a2-972c-ec6f4e766f6a.png)

**url-logo:** https://avatars.githubusercontent.com/u/5927154?s=280&v=4

#### question/feedback
- What do you all think of idea?
- Is it possible to use old logo jnode in jnode-revisited?

*Just a thought.*

**Comment by tripleo1** [comment, marked as answer]
Done

**Reply by ghost** [reply]
Hi tripleo1.

> Done

thank you for feedback.
